# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1346281340 14400
#      Wed Aug 29 19:02:20 2012 -0400
# Node ID e8abaaaa2d7d0763175568130d73fbe394a33ea0
# Parent  701532350420d2afaad569e7d65d813253ccd20f
avoid function defined but not used warning

* lo-cutils.c (gripe_missing_wait_macro): Declare inline.

diff --git a/liboctave/lo-cutils.c b/liboctave/lo-cutils.c
--- a/liboctave/lo-cutils.c
+++ b/liboctave/lo-cutils.c
@@ -74,17 +74,17 @@ octave_w32_library_search (HINSTANCE han
 #endif
 
 OCTAVE_API pid_t
 octave_waitpid (pid_t pid, int *status, int options)
 {
   return WAITPID (pid, status, options);
 }
 
-static void
+static inline void
 gripe_missing_wait_macro (const char *id, int status)
 {
   (*current_liboctave_warning_handler)
     ("%s always returns false in this version of Octave; status = %d",
      id, status);
 }
 
 OCTAVE_API int
